Update agent_integration.md (#23104)
DataDog/documentation: content/en/developers/integrations/agent_integration.md

  ```
</details>

> For installing your wheel to test in containerized environments: 

Author: ```suggestion
For installing your wheel to test in Kubernetes environments: 
```

> 1. Mount the whl into a initContainer

Reviewer: ```suggestion
1. Mount the `.whl` file into an initContainer.
```

> 2. Run the whl install in the init container

Reviewer: ```suggestion
2. Run the wheel install in the initContainer.
```

> 2. Run the whl install in a init container

Author: ```suggestion
2. Run the whl install in the init container
```

> 3. Mount the init container in the agent container

Author: ```suggestion
3. Mount the init container in the agent container while it's running
```

> 3. Mount the init container in the agent container while it's running

Reviewer: ```suggestion
3. Mount the initContainer in the Agent container while it's running.
```

> 3. Mount the init container in the agent container while it's running

Reviewer: We're gonna need some commands for these.

Author: I think this is something they configure, not something we instruct them to already have. Agent would be best to loop in for commands if we need them @hestonhoffman

> For customer install commands for both host and container environments, please see [here.][35]

Reviewer: ```suggestion
For customer install commands for both host and container environments, see the [Community and Marketplace Integrations documentation][35].
```
